test/unit/liquid/tags/active_docs_test.rb

```diff
@@ -14,7 +14,11 @@ class Liquid::Tags::ActiveDocsTest < ActiveSupport::TestCase
       match = params.match Liquid::Tags::ActiveDocs::Syntax
       assert match
       assert_equal match[1], expected[0]
-      assert_equal match[2], expected[1]
+      if expected[1]
+        assert_equal match[2], expected[1]
+      else
+        assert_nil match[2]
+      end
```

Comment:
:crying_cat_face:

Reply:
Hehe, do you have a better solution? :)

Reply:
If we can't know in prior what it would be, then no.